### DISCUSSION TITLE
Subscribing to local SB messages but not over SBN

### DISCUSSION THREAD

**Discussion by DavidKelly-MDA** [nasa/cFS#540] [Q&A] [answered]
Let's say we have two CPUs, CPU1 and CPU2, each with local SBs, and they are connected over SBN.
Can an App, APP1 on CPU1 subscribe to a message M1 on just CPU1 without also subscribing to M1 on CPU2?
Or does the SBN automatically require that the subscriptions are across all CPUs?

**Comment by skliper** [comment, marked as answer]
Have you tried `CFE_SB_SubscribeLocal`?  I think that's the intent of that API.
